training/model_management/src/azureml/model/mgmt/utils/common_utils.py

```diff
@@ -137,6 +137,7 @@ def tc_log(message):
     try:
         print(message)
         tc.track_event(name="FM_import_pipeline_debug_logs", properties={"message": message})
+        tc.flush()
```

Comment:
Could you please remove call to `flush`? Process exit should take care of it.

Reply:
it is not working, that is why added again